Yoast/wpseo-woocommerce
# Handle permalink changes

## Summary

This PR can be summarized in the following changelog entry:

* N/A - Clears the WooCommerce related indexables

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Change the WooCommerce permalink settings and verify in the database that the permalinks for the related indexables (products, attribute terms, product categories and product tags) are set to `NULL`

Fixes #501
